hack/verify-api-groups.sh

```diff
@@ -56,8 +56,10 @@ for register_file in "${register_files[@]}"; do
 		group_dirnames+=("${group_dirname}")
 		expected_install_packages+=("${package}")
 	else
-		version=$(echo "${group_dirname}" | cut -d/ -f2 -)
-		external_group_versions+=("${group_name}/${version}")
+		version=$(echo "${group_dirname}" | rev | cut -d/ -f1 | rev)
+		if [[ ${version} =~ v[0-9] ]]; then
+			external_group_versions+=("${group_name}/${version}")
+		fi
```

Comment:
This is trickier. This script works as-is in CI so I'm hesitant to make this change.  

I was able to repro the failures you're trying to fix by running this script with the repo in a path like `/foo/kubernetes`.  However, I wasn't ever hitting the "internal package" case above with or without your fix, and I should be.

This script expects the repo to live somewhere like `/foo/k8s.io/kubernetes`, which is a valid assumption to make in CI, but maybe no longer a valid assumption for developers. 

The fix you need to do is likely somewhere above, parsing out the `group_dirname` so that we correctly hit the internal case

Reply:
Understood.
I withdraw this modification because the script is working correctly on CI assumption.